src/SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests/Features/ConfirmYourEmployerSteps.cs

```diff
@@ -78,5 +110,31 @@ public void ThenTheLinkIsPointingToTheConfirmPage()
             var page = _context.ActionResult.LastPageResult;
             page.Model.Should().BeOfType<ConfirmYourEmployerModel>().Which.Backlink.Should().Be(_backlink);
         }
+
+        [Then(@"the user should be redirected back to the overview page")]
+        public void ThenTheUserShouldBeRedirectedBackToTheOverviewPage()
+        {
+            var redirect = _context.ActionResult.LastActionResult as RedirectToPageResult;
+            redirect.Should().NotBeNull();
+            redirect.PageName.Should().Be("Confirm");

```

Comment:
I like this, much better way of checking redirects than looking at the URL of the HttpResponse.